### ISSUE TITLE
SIM800 connection problem

### ISSUE THREAD

**Issue by sobiso** [alvarowolfx/asset-tracker-gcp-mongoose-os#1]
Have you had problems with internet connection?

[Jul 12 08:10:17.866] mgos_net_on_change_c PPP: ready, IP 10.161.111.224, GW 192.168.254.254, DNS 89.108.202.21
[Jul 12 08:10:17.870] net_cb               Net got IP address
[Jul 12 08:10:17.876] mgos_bt_net_ev       Network is up, disabling Bluetooth
[Jul 12 08:10:17.969] save_cfg             Saved to conf9.json
[Jul 12 08:10:17.976] mgos_mqtt_global_con MQTT connecting to iot.eclipse.org:1883
[Jul 12 08:10:18.899] mgos_sntp_query      SNTP query to time.google.com
[Jul 12 08:10:25.667] mgos_mqtt_ev         MQTT TCP connect ok (0)
[Jul 12 08:10:28.240] mg_resolve_async_eh  Failed to resolve 'time.google.com', server 89.108.202.21
[Jul 12 08:10:32.710] mgos_sntp_query      SNTP query to time.google.com
[Jul 12 08:10:35.266] mgos_mqtt_ev         MQTT CONNACK 0
[Jul 12 08:10:35.272] do_subscribe         Subscribing to 'esp32_45834C/rpc/#'
[Jul 12 08:10:35.278] do_subscribe         Subscribing to 'esp32_45834C/rpc'
[Jul 12 08:10:40.838] led_timer_cb         Tick uptimee: 61.46, RAM: 228196, 121276 free
[Jul 12 08:10:42.240] mg_resolve_async_eh  Failed to resolve 'time.google.com', server 89.108.202.21
[Jul 12 08:10:47.831] mgos_sntp_query      SNTP query to time.google.com
[Jul 12 08:10:57.280] mg_resolve_async_eh  Failed to resolve 'time.google.com', server 89.108.202.21

I tried few times (with different cards ) and i get maybe 3-4 times SNTP connection success

**Comment by alvarowolfx** [comment]
Yeah, I had some issues if I didn't use an Antenna for the SIM800L module and also saying inside my house the internet struggled to maintain the connection.

**Comment by sobiso** [comment]
You think that source of this problem is SIM800L or something with PPPoS lib ?

**Comment by alvarowolfx** [comment]
I think that is related with the network signal strength, because you were able to connect sometimes, but the network is not maintaining the connection, I had similar issues when I tried to use without a antenna or powering with the wrong voltage the SIM800l module.
